FIPS/pom.xml

```diff
@@ -57,6 +57,11 @@
 
   <dependencyManagement>
     <dependencies>
+      <dependency> <!-- commons-codec is not a direct dependency. Used by arrow-vector, httpclient, google-client-api -->
```

Comment:
Commons-codec has to be excluded from FIPS as well as regular file, since this issue popped up: https://github.com/snowflakedb/snowflake-jdbc/issues/831